service/frontend/accessControlledHandler.go

```diff
@@ -882,3 +1022,9 @@ func (a *AccessControlledWorkflowHandler) getMetricsScopeWithDomainName(
 ) metrics.Scope {
 	return a.GetMetricsClient().Scope(scope).Tagged(metrics.DomainTag(domainName))
 }
+
+func isManual(ctx context.Context) bool {
+	callContext := yarpc.CallFromContext(ctx)
+	caller := callContext.Caller()
+	return caller == "cadence-cli" || caller == "cadence-web" || caller == "cadence-ui"
```

Comment:
Could you add a comment above that sometime in the future we should also validate the caller.

Reply:
done